apps/snitch_core/lib/core/data/schema/state.ex

```diff
@@ -1,13 +1,16 @@
-defmodule Core.Snitch.State do
-  use Ecto.Schema
-  import Ecto.Changeset
-  alias Core.Snitch.Country
-  alias __MODULE__, as: State
+defmodule Snitch.Data.Schema.State do
+  @moduledoc """
+  Models a State
+  """
+
+  use Snitch.Data.Schema
+  alias Snitch.Data.Schema.Country
+  alias __MODULE__
```

Comment:
No use of `State`, so no need of this alias.

Reply:
used in the changeset.